<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceivingWindow</class>
 <widget class="QMainWindow" name="ReceivingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>49</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Описание</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="productDescription">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="productCost">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоимость</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="productAmmount">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Количество</string>
    </property>
   </widget>
   <widget class="QComboBox" name="productWarehouse">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Склад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goBackButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QComboBox" name="productInput">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addReceivingButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Получить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>428</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
